demo_modules/slideshow/slideshow.js

```diff
@@ -180,18 +189,22 @@ class LoadFromDriveClientStrategy extends ClientLoadStrategy {
       .then((resp) => resp.blob())
       .then((blob) => URL.createObjectURL(blob))
       .then((url) => {
-        var img = document.createElement('img');
-        img.style.position = 'absolute';
-        img.style.top = 0;
-        img.style.left = 0;
-        img.style.width = '100%';
-        img.style.height = '100%';
-        img.src = url;
-        return img;
+        return new Promise((resolve, reject) => {
+          var img = document.createElement('img');
+          img.src = url;
+          // Don't report that we've loaded the image until onload fires.
+          img.onload = () => resolve(img);
+          img.onerror = (e) => reject(e);
```

Comment:
Are you sure onerror passes an error rather than an event as the first arg or are you expecting to reject with an error event?

Also, maybe img.addEventListener('load', () => resolve(img)) and img.addEventListener('error', () => reject(new Error(<some message>)) would be better than using onerror/onload.  I'll leave it up to you.